.typokana_ignore

```diff
@@ -0,0 +1,142 @@
+nodoc
+Luhn
+luhn
+gregorian
+kcp
+Prefilled
+prefill
+docs.adyen.com
+Cartebancaire
+cartebancaire
+americanExpress
+Forbrugsforeningen
+visadankort
+dankort
+Sdk
+sdk
+bcmc
+binlookup
+plcc
+uatp
+nyce
+Dejeneur
+cencosud
+accel
+jcb
+hipercard
+codensa
+argencard
+maestrouk
+karenmillen
+hipercard
+visaalphabankbonus
+mcalphabankbonus
+Indomaret
+interac
+Dotpay
+Qiwi
+qiwi
+qiwiwallet
+boletobancario
+applepay
+wechatpay
+mbway
+bancontact
+econtext
+Bancairo
+directdebit
+sepa
+oxxo
+genericgiftcard
+openbanking
+molpay
+paywithgoogle
+afterpay
+androidpay
+amazonpay
+santander
+hpp
+Doku
+Boleto
+Multibanco
+giropay
+Palx
+klarna
+ebanking
+elodebit
+swiftlint
+Hashable
+Codable
+Hidable
+rhs
+lhs
+Unkeyed
+psp
+Iterable
+Finalizable
+GitHub
+weixin
+Eldoheiri
+Abramichev
```

Comment:
Those we can probably replace with "standard" headers?

Reply:
its not for headers, those names are used in unit tests :-D

Reply:
ok 🤔